src/main.rs

```diff
@@ -28,6 +28,13 @@ fn get_config_file_path(dir: &Path, config_path: &Path) -> (PathBuf, PathBuf) {
 
 fn main() {
     let cli = Cli::parse();
+
+    if let Command::Completion { shell } = &cli.command {
```

Comment:
can that be moved in the match below?

Reply:
I wanted to make sure that `zola completion` would never fail, but at second glance it seems there wouldn't be much difference with the match version

Reply:
Still, to make sure completion just works I'd keep it as it is

Reply:
I'd still prefer it in the match, I don't mind a user passing an invalid flag erroring

Reply:
Done, then 😄